modules/windows-containers-release-notes-limitations.adoc

```diff
@@ -21,9 +21,11 @@ Note the following limitations when working with Windows nodes managed by the WM
 ** link:https://docs.redhat.com/en/documentation/cost_management_service/1-latest[Red Hat Insights cost management]
 ** link:https://developers.redhat.com/products/openshift-local/overview[Red Hat OpenShift Local]
 
-* Windows nodes do not support workloads created by using deployment configs. You can use a deployment or other method to deploy workloads.
+* The WMCO does not support OVN-Kubernetes without hybrid networking or OpenShift SDN.
```

Comment:
I'd rather use this as a requirement, not a limitation. 

@openshift/openshift-team-windows-containers   WDYT?